packages/docs-components/src/knobs/KnobProvider.tsx

```diff
@@ -12,13 +12,14 @@ const KnobProvider: React.FunctionComponent<KnobProviderProps> = props => {
   const [knobs, setKnobs] = React.useState<KnobSet>({})
 
   const registerKnob = (knob: KnobDefinition) => {
-    if (process.env.NODE_ENV !== 'production') {
-      if (knobs[knob.name]) {
-        throw new Error(`Knob with name "${knob.name}" has been already registered`)
+    setKnobs(prevKnobs => {
```

Comment:
fix for the problem with using a different names for the same knob, thx @layershifter

Reply:
This breaks existing docsite pages with knobs (`Button`, `Menu`):
![image](https://user-images.githubusercontent.com/9615899/58277199-75b86d80-7d99-11e9-9b30-9eff47bfdeeb.png)

Reply:
reverted so that I get unblocked, thanks, @layershifter let's tackle this fix in separate PR as the one proposed introduces the issue @miroslavstastny is mentioning